main.js

```diff
@@ -0,0 +1,111 @@
+// Ex 1: Thuế và lương ròng
+const getSalery = (salery) => {
+    if(salery >= 15000000){
+        console.log('Thuế thu nhập =',salery*30/100);
```

Comment:
Nhìn kỹ cách đặt dấu cách và sửa hết toàn bộ bên dưới
```suggestion
        console.log('Thuế thu nhập =', salary * 0.3);
```